src/python/grpcio/grpc/experimental/aio/_interceptor.py

```diff
@@ -504,6 +551,149 @@ def time_remaining(self) -> Optional[float]:
         raise NotImplementedError()
 
 
+class InterceptedStreamUnaryCall(_InterceptedUnaryResponseMixin,
+                                 InterceptedCall, _base_call.StreamUnaryCall):
+    """Used for running a `StreamUnaryCall` wrapped by interceptors.
+
+    For the `__await__` method is it is proxied to the intercepted call only when
+    the interceptor task is finished.
+    """
+
+    _loop: asyncio.AbstractEventLoop
+    _channel: cygrpc.AioChannel
+    _write_to_iterator_async_gen: Optional[AsyncIterable[RequestType]]
+    _write_to_iterator_queue: Optional[asyncio.Queue]
+
+    _FINISH_ITERATOR_SENTINEL = tuple()
```

Comment:
nit: Should we use `aio.EOF`?

Reply:
Don't you think that `EOF` is more a public object that is used for indicating and transmitting the end of an iterator, while here is purely an internationally that is used for stopping the iterator? Using the `EOF` might confuse further maintainers.

Reply:
Sounds good. I'm okay with either option.

Reply:
Oks